src/eq_schema/Question.test.js

```diff
@@ -313,10 +313,10 @@ describe("Question", () => {
       expect(question).toHaveProperty("mandatory");
     });
 
-    it("should inherit mandatory property from mutually exclusive answer", () => {
+    it("should not set mandatory on exclusive child answers", () => {
```

Comment:
The title of this test is a bit misleading. The question has `mandatory: false`. But the test says that child answers should not have mandatory set. Do we need another test to assert both?

Reply:
Ah yeah that's bad wording, think when I was testing this the answers needed a mandatory property, but it had to be false. I'll change the test title to something a bit better.